docs/source/tools/sandbox.rst

```diff
@@ -112,11 +117,15 @@ where
 
 The ``public`` claim is implicitly held by anyone bearing a valid JWT (even without being an admin or being able to act or read on behalf of any party).
 
-Generating tokens
-=================
+Generating JWT tokens
```

Comment:
Please choose between either JWT or tokens. The _T_ in _JWT_ stands for token. :wink:

Reply:
```suggestion
Generating JSON Web Tokens (JWT)
```

This would also be an alternative (inspired by the intro text on https://jwt.io/)

Reply:
updated based on @rautenrieth-da suggestion